uui-components/src/inputs/BaseTimePicker.tsx

```diff
@@ -14,6 +14,9 @@ export interface BaseTimePickerProps extends IEditable<TimePickerValue | null>,
         input?: IHasRawProps<React.HTMLAttributes<HTMLDivElement>>['rawProps'];
         body?: IHasRawProps<React.HTMLAttributes<HTMLDivElement>>['rawProps'];
     };
+    /** Styles for input and body components */
```

Comment:
add the comment for each prop and let's make it more specific, look to the description of general 'cx' props and use the same structure and wordings.

Reply:
done